events/helper/inbound_connection.go

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+Copyright (C) 2024 The Falco Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/falcosecurity/event-generator/events"
+)
+
+var _ = events.Register(InboundConnection)
+
+func InboundConnection(h events.Helper) error {
+	address, _ := getRandomLocalAddress()
+	listener, err := net.Listen("tcp", address)
+	if err != nil {
+		return err
+	}
+	fmt.Println("Server started on", address)
+	defer listener.Close()
+	fmt.Println("Server closed on", address)
```

Comment:
```suggestion
	fmt.Println("Server closed on", address)
```

This is not accurate since the sever will be actually closed after func `listener.Close()` returns. I just propose to remove this line.